### ISSUE TITLE
Helm chart contains wrong secret for loki deployment

### ISSUE THREAD

**Issue by pavel-khritonenko** [grafana/loki#531]
**Describe the bug**

Rendered helm template to the local files, deployed to Kubernetes cluster, pod `default-loki-<rs>-<pod>` is failing with the error: 

    level=error ts=2019-04-30T13:05:25.898083784Z caller=main.go:46 msg="error loading config" filename=/etc/loki/loki.yaml err="yaml: unmarshal errors:\n  line 3: field chunk_block_size not found in type ingester.Config"

**To Reproduce**
1. Cloned repository (commit `eb43bb6`)
2. Rendered both `loki` and `promtail` to static manifests using `helm template ...`
3. Deployed to the cluster using `kubectl apply -f charts/ --recursive`

**Expected behavior**
Everything started successfully and works fine.

**Environment:**
 - Infrastructure: Kubernetes
 - Deployment tool: Helm (rendered static manifests using `helm template`)

**Screenshots, promtail config, or terminal output**

    level=error ts=2019-04-30T13:05:25.898083784Z caller=main.go:46 msg="error loading config" filename=/etc/loki/loki.yaml err="yaml: unmarshal errors:\n  line 3: field chunk_block_size not found in type ingester.Config"

**Workaround:**
Edit secret and remove `chunk_block_size` from the `ingester` section

**Comment by akoenig** [comment]
We can confirm the problem when deploying the helm chart via:

```
helm repo add loki https://grafana.github.io/loki/charts && helm repo update

helm upgrade --install loki loki/loki
```

**Comment by steven-sheehy** [comment]
Thanks for reporting. Looks like a regression due to #528. You can use the previous version until this is fixed (`0.7.2`). @daixiang0 Mind addressing this?

**Comment by steven-sheehy** [comment]
Merged a fix for this. There's no new chart version since it didn't require any chart changes. To get the fix you can just delete your loki pod and it should pull the latest since `imagePullPolicy: Always`.
